admin/common_auth/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class AdminUser(models.Model):
```

Comment:
All users are "admin users" technically so I don't think we should make everyone `AdminUser` but rather just a `User` and the admin is implied by the site it is on. Take a look through https://docs.djangoproject.com/en/1.8/topics/auth/default/ for some information in that direction. I think we might be able to authenticate using built-in Django authentication.